.eslintrc.identical.keys.cjs

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path');
```

Comment:
Are we have any reason cjs here?

Reply:
yeah, eslint need this, if .mjs or .js if fails with next error 
```
Oops! Something went wrong! :(

ESLint: 8.24.0

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/pavelropakov/Documents/projects/github/disaster-ninja-fe/.eslintrc.identical.keys.js from /Users/pavelropakov/Documents/projects/github/disaster-ninja-fe/node_modules/@eslint/eslintrc/dist/eslintrc.cjs not supported.
Instead change the require of .eslintrc.identical.keys.js in /Users/pavelropakov/Documents/projects/github/disaster-ninja-fe/node_modules/@eslint/eslintrc/dist/eslintrc.cjs to a dynamic import() which is available in all CommonJS modules.